index.html

```diff
@@ -16,6 +18,11 @@
 
   <body>
     <header>
+      <hgroup>
```

Comment:
I think the `hgroup` has been [dropped from html5](http://www.sitepoint.com/html5-hgroup-element-dropped/):

> Do you use the hgroup element in your mark-up? It’s probably best not to — the tag is being dropped from the HTML5 specification.